cmd/consumer/main.go

```diff
@@ -57,7 +57,7 @@ func consumeEvent(event cloudevents.Event) error {
 	}
 	// RequestURI must be unset for client.Do(req)
 	req.RequestURI = ""
-	req.Header.Del("Prefer") // We do not want to make this request as async
+	req.Header.Set("Prefer", "respond-sync") // We do not want to make this request as async
```

Comment:
Typo here? "Mark as". Also is this the right way round, seems like we "do" not "do not"?

Reply:
I introduced the `respond-sync` header to mark the request coming from the consumer so that the ingress rules know to send it directly to the service. Once a request gets to the consumer, we don't want to make it asynchronously from the consumer's point of view.

Reply:
Should those be consts?